ansible/roles/satellite-configuration/tasks/01_manifest.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: fetching list of existing subscriptions
+  shell: hammer --output cve subscription list --organization "{{ORG}}"
+  register: subscription_list
+  ignore_errors: yes
+  tags:
+    - satellite_configuration
+    - manifest
+- debug: var=subscription_list
+- name: Creating list fact of existing subscriptions
+  set_fact:
+    list_of_existing_subscriptions: "{{ list_of_existing_subscriptions + [ item.split(':').1.lstrip(' ') ] }}"
+  loop: "{{ subscription_list.stdout_lines }}"
```

Comment:
what happen to this task and the following if `fetching list of existing subscriptions` fails ?
i'm asking because you added `ignore_errors: yes` to `fetching list of existing subscriptions`.

Reply:
Fixed